src/ray/raylet/scheduling_resources.h

```diff
@@ -215,6 +236,26 @@ class ResourceIds {
   /// \return A human-readable string representing the object.
   std::string ToString() const;
 
+  /// \brief Increase resource capacity by the given amount. This may throw an error if
+  /// decrement is more than currently available resources.
+  ///
+  /// \param new_capacity double of new capacity
+  /// \return Void.
+  void UpdateCapacity(double new_capacity);
+
+  /// \brief Increase resource capacity by the given amount.
+  ///
+  /// \param increment_quantity A double of how many unit resources to add.
+  /// \return Void.
+  void IncreaseCapacity(double increment_quantity);
```

Comment:
`IncreaseCapacity` and `DecreaseCapacity` should be private, and `UpdateCapacity` is the only public one.